src/OrchardCore/OrchardCore.Abstractions/Extensions/Features/ITypeFeatureProvider.cs

```diff
@@ -10,8 +10,39 @@ namespace OrchardCore.Environment.Extensions
     /// </summary>
     public interface ITypeFeatureProvider
     {
+        /// <summary>
+        /// Gets the extension for the specified dependent type.
```

Comment:
Add a `<remark>` about why the first one is interesting, why do we have this separate methods (because otherwise it's curious: "what's the significance of the first feature?").

Reply:
Done.